proto/substrait/algebra.proto

```diff
@@ -136,14 +136,42 @@ message ReadRel {
       message ArrowReadOptions {}
       message OrcReadOptions {}
       message DwrfReadOptions {}
+      message DelimeterSeparatedTextReadOptions {
+        // Delimiter separated files may be compressed.  The reader should
+        // autodetect this and decompress as needed.
+
```

Comment:
one other question here. Should "trim" be an option. For example, will these two records return different values. 

```
A, B, C
A,B,C
```
If so, it that typically desired? My recollection is many readers will trim whitespace but I could be misremembering.

Reply:
Spark, DuckDb, and Pandas do not have such an option.  I think we should avoid anything that isn't already implemented somewhere.  We can always add more options later without introducing a backwards incompatible change.

Reply:
I'm okay without the option. What is the specified behavior? Trim or not trim?

Reply:
The standard behavior is no trimming.

Reply:
ok, captured here. ideally would make itself into docs at some point.